pkg/experiment/sensitivity/factory.go

```diff
@@ -121,6 +125,12 @@ type ExecutorFactoryFunc func(isolation.Decorators) (executor.Executor, error)
 
 // Create returns aggressor of chosen type using provided executorFactor to create executor.
 func (f AggressorFactory) Create(name string, executorFactory ExecutorFactoryFunc) (executor.Launcher, error) {
+
+	// Zero-value sensitivity.LauncherSessionPair represents baselining.
+	if name == NoneAggressorID {
+		return nil, nil
```

Comment:
@ppalucki, @skonefal - I think that having implicit `if` in the experiment code is clearer. As @ppalucki suggests - it can be fixed when we get rid of current Snap integration implementation.